### PR TITLE
Allow contacts to be created via contact form launch

### PR DESCRIPTION
**Pull request by xyuka**
create a contact with the corresponding new to-be-saved user data and display in a contactsUI view

### CODE REVIEW

File: android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java [hunk 1]
```diff
@@ -130,9 +131,140 @@ public void run() {
...
+        intent.putExtra(ContactsContract.Intents.Insert.NAME, displayName);
+        intent.putParcelableArrayListExtra(ContactsContract.Intents.Insert.DATA, contactData);
+        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+
+        Context context = getReactApplicationContext();
+        context.startActivity(intent);
+
+    }
+    /*
+         * Adds contact to phone's addressbook
```
Comment (morenoh149) [inline android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java]: If you place this on the same indentation as it was the diff will not pick it up. Please undo the additional indentation on the comment.

**Comment by xyuka** [comment]
I have add some description about `createContact` on README and revise the indentation on the comment. please check. thanks!

**Comment by morenoh149** [comment]
Looks good. Will review when I have some time. If you have more time please add an integration test over at https://github.com/morenoh149/react-native-contacts-test/

**Comment by morenoh149** [comment]
I feel like this should be called `openContactForm` instead. Please change the name.

**Comment by xyuka** [comment]
I have update the method name to `openContactForm`. thanks.

**Comment by xyuka** [comment]
Hi updated the readme already!~ thanks.

**Comment by morenoh149** [comment]
@xyuka thanks!

**Pull request merged**